core/src/test/scala/org/apache/spark/sql/delta/DeleteSQLSuite.scala

```diff
@@ -97,3 +97,22 @@ class DeleteSQLNameColumnMappingSuite extends DeleteSQLSuite
   }
 
 }
+
+class DeleteSQLWithDeletionVectorsSuite extends DeleteSQLSuite with DeletionVectorsTestUtils {
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    enableDeletionVectorsForDeletes(spark)
+  }
+
+  override def excluded: Set[String] = super.excluded ++ Set(
+    "data and partition columns - Partition=true Skipping=false",
+    "data and partition columns - Partition=false Skipping=false",
+    "nested schema pruning on data condition")
+
+  // This works correctly with DVs, but fails in classic DELETE.
```

Comment:
Could you please create an issue for this?

Reply:
Sure, will do.